doc/conf.py

```diff
@@ -83,3 +83,80 @@
+    ("man/ipa-cert-fix.1", "ipa-cert-fix", "Renew expired certificates",
+     None, 1),
+    ("man/ipa-certupdate.1", "ipa-certupdate",
+     "Update local IPA certificate databases with", None, 1),
+    ("man/ipa-client-automount.1", "ipa-client-automount",
+     "Configure automount and NFS for IPA", None, 1),
+    ("man/ipa-client-install.1", "ipa-client-install",
+     "Configure an IPA client", None, 1),
+    ("man/ipa-client-samba.1", "ipa-client-samba",
+     "Configure Samba file server on an IPA client", None, 1),
+    ("man/ipa-compat-manage.1", "ipa-compat-manage",
+     "Enables or disables the schema compatibility plugin", None, 1),
+    ("man/ipa-crlgen-manage.1", "ipa-crlgen-manage",
+     "Enables or disables CRL generation", None, 1),
+    ("man/ipa-csreplica-manage.1", "ipa-csreplica-manage",
+     "Manage an IPA CS replica", None, 1),
+    ("man/ipa-dns-install.1", "ipa-dns-install",
+     "Add DNS as a service to an IPA server", None, 1),
+    ("man/ipa-getkeytab.1", "ipa-getkeytab",
+     "Get a keytab for a Kerberos principal", None, 1),
+    ("man/ipa-join.1", "ipa-join",
+     "Join a machine to an IPA realm and get a keytab for the host "
+     "service principal", None, 1),
+    ("man/ipa-kra-install.1", "ipa-kra-install",
+     "Install a KRA on a server", None, 1),
+    ("man/ipa-ldap-updater.1", "ipa-ldap-updater",
+     "Update the IPA LDAP configuration", None, 1),
+    ("man/ipa-managed-entries.1", "ipa-managed-entries",
+     "Enables or disables the schema Managed Entry", None, 1),
+    ("man/ipa-nis-manage.1", "ipa-nis-manage",
+     "Enables or disables the NIS listener plugin", None, 1),
+    ("man/ipa-otptoken-import.1", "ipa-otptoken-import",
+     "Imports OTP tokens from RFC 6030 XML file", None, 1),
+    ("man/ipa-pkinit-manage.1", "ipa-pkinit-manage",
+     "Enables or disables PKINIT", None, 1),
+    ("man/ipa-replica-conncheck.1", "ipa-replica-conncheck",
+     "Check a replica-master network connection before", None, 1),
+    ("man/ipa-replica-install.1", "ipa-replica-install",
+     "Create an IPA replica", None, 1),
+    ("man/ipa-replica-manage.1", "ipa-replica-manage",
+     "Manage an IPA replica", None, 1),
+    ("man/ipa-restore.1", "ipa-restore", "Restore an IPA master", None, 1),
+    ("man/ipa-rmkeytab.1", "ipa-rmkeytab",
+     "Remove a kerberos principal from a keytab", None, 1),
+    ("man/ipa-server-certinstall.1", "ipa-server-certinstall",
+     "Install new SSL server certificates", None, 1),
+    ("man/ipa-server-install.1", "ipa-server-install",
+     "Configure an IPA server", None, 1),
+    ("man/ipa-server-upgrade.1", "ipa-server-upgrade",
+     "upgrade IPA server", None, 1),
+    ("man/ipa-winsync-migrate.1", "ipa-winsync-migrate",
+     "Seamless migration of AD users created by winsync", None, 1),
+    ("man/ipa.1", "ipa", "IPA command-line interface", None, 1),
+    ("man/ipactl.8", "ipactl", "IPA Server Control Interface", None, 8),
+]
```

Comment:
Is there a way to auto-discover these? I fear it will get missed if/when a new man page is added (heck, we often forget the man page updates too).

Reply:
There is no easy way to detect this. :( I could add a comment to each Makefile so it gets detected in code review.

Please note that this data structure is used for ``make man`` to turn the restructured text files back to groff man pages. In the long run I hope that we can generate man pages from RST instead of the other way around.